Leanplum-SDK/Classes/Managers/LPFileManager.m

```diff
@@ -834,4 +834,26 @@ + (void)enableResourceSyncingWithInclusions:(NSArray *)inclusions
     }
 }
 
++ (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
```

Comment:
So this is a new feeature or something? Its not used internally

Reply:
Its a system feature to skip the backup of file/folder. I'm not sure if there are any possible tests here to add.

Reply:
So a customer is requesting this feature? Also, test cases are for code coverage. This is public method. So, you can just write a very simple test cases for addSkipBackupAttributeToItemAtPath method which returns true